docs/overview/creating_effects.md

```diff
@@ -58,7 +58,7 @@ Scala's standard library contains a number of data types that can be converted i
 An `Option` can be converted into a ZIO effect using `ZIO.fromOption`:
 
 ```scala mdoc:silent
-val zoption: IO[Unit, Int] = ZIO.fromOption(Some(2))
+val zoption: IO[Option[Nothing], Int] = ZIO.fromOption(Some(2))
 ```
 
 The error type of the resulting effect is `Unit`, because the `None` case of `Option` provides no information on why the value is not there. You can change the `Unit` into a more specific error type using `ZIO#mapError`:
```

Comment:
Does this paragraph need to be updated in lieu of the error channel now being defined as `Option[Nothing]` instead of `Unit`?